open-api/rest-catalog-open-api.yaml

```diff
@@ -1482,6 +1490,24 @@ components:
       explode: false
       example: "vended-credentials,remote-signing"
 
+    page-token:
+      name: pageToken
+      in: query
+      required: false
+      allowEmptyValue: true
+      schema:
+        $ref: '#/components/schemas/PageToken'
+
+    page-size:
+      name: pageSize
+      in: query
+      description:
+        For servers that support pagination, this signals an upper bound of the number of results that a client will receive. 
+        For servers that do not support pagination, clients may receive results larger than the indicated `pageSize`.
+      required: false
+      schema:
+        type: integer
```

Comment:
Taking a look back after the #9782 should we enforce a minimum number here like 1, so we can end up with validation like this:
 
<img width="807" alt="Screenshot 2024-02-22 at 12 07 24 PM" src="https://github.com/apache/iceberg/assets/19402187/27b1b01d-f49e-4aac-b1d7-18b960da37aa">

Reply:
Makes sense thanks @geruh